Documentation/contributor-guide/community-membership.md

```diff
@@ -115,6 +115,14 @@ depend on to make decisions in the best interest of the project in a consistent
 - Subscribed to [etcd-maintainers@googlegroups.com]
```

Comment:
Removing it from `Requirements` and adding to `Onboarding process` might be good?

Reply:
Having them combined like for members is based on K8s community document. I think it makes sense because it's a requirement to become a maintainer.

Reply:
I see, let me update `Requirements` for describing onboarding process.